content/en/platform/corda/5.0/deploying-operating/deploying/deploying.md

```diff
@@ -310,6 +323,42 @@ username:
   value: <USERNAME>
 ```
 
+### Encryption
+
+The Corda configuration system allows for any string configuration value to be marked as “secret”. This includes values passed dynamically using the REST API and also those defined in a manual deployment configuration. For more information see, [Configuration Secrets]({{< relref "../config/secrets.md" >}}).
+
+#### Default Secrets Service
+
+The [Corda default secrets lookup service]({{< relref "../config/secrets.md#default-secrets-service" >}}) uses a salt and passphrase specified in the deployment configuration. Specify these as follows:
+
+```yaml
+config:
+   encryption:
+      salt: <SALT>
+      passphrase: <PASSPHRASE>
```

Comment:
Actually, this changed a long time ago. They are now specified in a similar manner to other credentials in the configuration YAML. They can be passed explicitly:
```
config:
   encryption:
      salt:
        value: <SALT>
      passphrase: 
        value: <PASSPHRASE>
```
Or the preferred approach, via Kubernetes secret:
```
config:
   encryption:
      salt:
        valueFrom:
          secretKeyRef:
            name: <SALT_SECRET_NAME>
            key: <SALT_SECRET_KEY>
      passphrase: 
        valueFrom:
          secretKeyRef:
            name: <PASSPHRASE_SECRET_NAME>
            key: <PASSPHRASE_SECRET_KEY>
```
In most places in the doc we just show the latter and then I thought (though I can't find it now), that there was a comment somewhere saying that wherever you see `valueFrom` you can also use `value` to specify it inline.

Reply:
I've just added the latter for now so as not to over-complicate things at this late stage. Probably something we should come back to post-release?